R/callOpenTiles.R

```diff
@@ -620,4 +637,4 @@ setMethod(
     )
   )
```

Comment:
Save tileResults to outDir for shareMOCHA

Reply:
@markphillippebworth If we save a tileResults.rds in the directory alongside coverage files, the functions shareMOCHA (now pack/unpackMOCHA) should either only accept and return the tileResults or only the sample-tile matrix. Currently pack/unpackMOCHA can take either and will return either, which can be confusing.

Reply:
-- we need to add a line of code to save tileResults as an HDF5 format automatically with call open tiles.
shareMOCHA should save SampleTimeMatrix as an HDF5
unpackMOCHA loads that HDF5 back in. 
loadTileResults should use a SampleTileMatrix to load the tileResults into memory from the HDF5

Reply:
Save HDF5 instead of RDS